app/gui/view/component-browser/component-list-panel/Cargo.toml

```diff
@@ -9,6 +9,7 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 enso-frp = { path = "../../../../../lib/rust/frp" }
+ensogl-component = { path = "../../../../../lib/rust/ensogl/component/" }
```

Comment:
The tooltip is also a separate crate. So we should do one of
1. Get the tooltip directly (avoiding compiling-in intermediate crates)
2. import ensogl-component and remove other dependencies from component directory, as they should be included in ensogl-component as well.